www-client/qupzilla/qupzilla-9999.ebuild

```diff
@@ -27,7 +27,7 @@ fi
 
 DESCRIPTION="A cross-platform web browser using QtWebEngine"
 HOMEPAGE="http://www.qupzilla.com/"
-SRC_URI+=" https://github.com/QupZilla/${PN}-plugins/archive/${PLUGINS_HASH}.tar.gz -> ${PN}-plugins-${PLUGINS_VERSION}.tar.gz"
+SRC_URI+="https://github.com/QupZilla/${PN}-plugins/archive/${PLUGINS_HASH}.tar.gz -> ${PN}-plugins-${PLUGINS_VERSION}.tar.gz"
```

Comment:
As I thought, thanks.

Should I drop the change from this pull request though, since it's not exactly fixing anything for the time being?